docs/fieldnorm/soups.md

```diff
@@ -0,0 +1,13 @@
+# SOUPS - Software of Unknown Provenance
+
+Software of unknown provenance (SOUPS) is a software item that is already developed, generally available and not developed for the purpose of being incorporated into the bioinformatic workflow, or a software item for which adequate software documentation is not available. A procedure should be documented about how and where SOUPS are registered.
+
+For each SOUP the following items should be registered:
+
+- Name: Name of SOUP.
+- Reference: Link (URL) to soup (source code / publication).
+- Intended purpose: Description of the original purpose of the SOUP.
+- Used according to the intended purpose: yes/no, if no, document for which purpose the SOUP is used.
+- Validation required: yes/no, often this is part of the validation of the (workflow) release itself. When using the SOUP outside of its intended purpose you need a proper validation to show that it works.
+- Version controlled: yes/no, is the SOUP developed under version control or not.
```

Comment:
It is not always possible to know if the SOUP under version control if it is from commercial parties. Is this problematic?

Reply:
not really, as longs as its clear which version of the software you're using it should be fine.